doc/nrf/gs_recommended_versions.rst

```diff
@@ -0,0 +1,187 @@
+           - |cmake_recommended_ver_win10|
+         * - dtc
+           - |dtc_min_ver|
+           - |dtc_recommended_ver_win10|
+         * - git
+           -
+           - |git_recommended_ver_win10|
+         * - GNU Arm Embedded Toolchain
+           - |gnuarmemb_min_ver|
+           - |gnuarmemb_recommended_ver_win10|
+         * - gperf
+           - |gperf_min_ver|
+           - |gperf_recommended_ver_win10|
+         * - ninja
+           - |ninja_min_ver|
+           - |ninja_recommended_ver_win10|
+         * - Python
+           - |python_min_ver|
+           - |python_recommended_ver_win10|
+         * - SEGGER Embedded Studio (Nordic Edition)
+           - |ses_min_ver|
+           - |ses_recommended_ver_win10|
+
+   .. group-tab:: Linux
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Tool
+           - Minimum version
+           - Tested version
+         * - ccache
+           -
+           - |ccache_recommended_ver_linux|
+         * - CMake
+           - |cmake_min_ver|
+           - |cmake_recommended_ver_linux|
+         * - dfu_util
+           -
+           - |dfu_util_recommended_ver_linux|
+         * - dtc
+           - |dtc_min_ver|
+           - |dtc_recommended_ver_linux|
+         * - git
+           -
+           - |git_recommended_ver_linux|
+         * - GNU Arm Embedded Toolchain
+           - |gnuarmemb_min_ver|
+           - |gnuarmemb_recommended_ver_linux|
+         * - gperf
+           - |gperf_min_ver|
+           - |gperf_recommended_ver_linux|
+         * - ninja
+           - |ninja_min_ver|
+           - |ninja_recommended_ver_linux|
+         * - Python
+           - |python_min_ver|
+           - |python_recommended_ver_linux|
+         * - SEGGER Embedded Studio (Nordic Edition)
+           - |ses_min_ver|
+           - |ses_recommended_ver_linux|
+
+   .. group-tab:: macOS
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Tool
+           - Minimum version
+           - Toolchain manager version
+         * - CMake
+           - |cmake_min_ver|
+           - |cmake_recommended_ver_darwin|
+         * - dtc
+           - |dtc_min_ver|
+           - |dtc_recommended_ver_darwin|
+         * - git
+           -
+           - |git_recommended_ver_darwin|
+         * - GNU Arm Embedded Toolchain
+           - |gnuarmemb_min_ver|
+           - |gnuarmemb_recommended_ver_darwin|
+         * - gperf
+           - |gperf_min_ver|
+           - |gperf_recommended_ver_darwin|
+         * - ninja
+           - |ninja_min_ver|
+           - |ninja_recommended_ver_darwin|
+         * - Python
+           - |python_min_ver|
+           - |python_recommended_ver_darwin|
+         * - SEGGER Embedded Studio (Nordic Edition)
+           - |ses_min_ver|
+           - |ses_recommended_ver_darwin|
+
+
+Required Python dependencies
+############################
+
+The following table shows the Python packages that are required for working with |NCS| |version|.
```

Comment:
```suggestion
The following table shows the Python packages that are required for working with |NCS| v|version|.
```

Reply:
I included this, and gets:
> v|version|

cause it don't replace correctly in that case.

but if I do:
```
... working with |NCS| v |version|.
```

then I get:
>  ... with nRF Connect SDK v 1.3.99. 

with an extra space. How can I get both `|version|` correctly replace AND the `v` next to it like: `v1.3.99` ?

Reply:
seems `ltrim` can do the trick, but should go together with the `|version|` definition.
https://docutils.sourceforge.io/docs/ref/rst/directives.html#replace

> ltrim : flag
>    Whitespace to the left of the substitution reference is removed.

Reply:
We could also just define |version| as "v1.3.99", but apparently that definition is implicit in the conf file ...
@utzig Any idea?

Reply:
try `v\ |version|`

Reply:
`\ ` will give you a space.
So won't help.

Reply:
![backslash](https://user-images.githubusercontent.com/107070/94264328-e4b55180-ff0c-11ea-8a63-9187b310c5c5.jpg)

I just happen to have **tried** it. Do you see a space in picture? Because I sure don't! For reference: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#escaping-mechanism

`In non-URI contexts, backslash-escaped whitespace characters are removed from the document. This allows for character-level inline markup.`

Reply:
strange, when I tried it locally yesterday then I still got a space.

Reply:
> strange, when I tried it locally yesterday then I still got a space.

It could be that I am already with newer documentation build tools, specifically docutils==0.16.0, and for some reason it did not work with previous versions, but in that case it will soon.

Reply:
> I just happen to have tried it.

so did I yesterday, but still got a space.
Have tried again now, and this time there is no space :)

Thanks.